cgmanifest.json

```diff
@@ -6,7 +6,7 @@
 				"git": {
 					"name": "abseil",
 					"repositoryUrl": "https://github.com/abseil/abseil-cpp",
-					"commitHash": "aa844899c937bde5d2b24f276b59997e5b668bde"
+					"commitHash": "b832dce8489ef7b6231384909fd9b68d5a5ff2b7"
```

Comment:
It would be great if you can help us updating this to the new `Pod` type. Here's an example:

```json
		{
			"Component": {
				"Type": "Pod",
				"Pod": {
					"Name": "abseil",
					"Version": "0.20200225.0"
				}
			}
		},
```

Thanks a lot in advance!

Reply:
Should all the components that are currently type `git` be updated to this new `Pod` type? What about the `other` type, do these stay the same?